.github/workflows/docker-ci.yml

```diff
@@ -0,0 +1,54 @@
+name: Docker CI
+
+on:
+  push:
+    tags:
+      - '*'
+    branches:
+      - master
+      - 'auto-cadence-upgrade/**'
+      - staging
+      - trying
+      - 'feature/**'
+      - 'v[0-9]+.[0-9]+'
+  pull_request:
+    branches:
+      - master
+      - 'auto-cadence-upgrade/**'
+      - 'feature/**'
+      - 'v[0-9]+.[0-9]+'
+
+env:
+  CONTAINER_REGISTRY: gcr.io/dapper-flow
+
+jobs:
+  build:
+    runs-on: ubuntu-latest
+    permissions:
+      contents: read
+      packages: write
+      id-token: write
+
+    steps:
+      - name: Checkout repository
+        uses: actions/checkout@v2
+      
+      - name: Login to GCR
+        uses: docker/login-action@v1
+        with:
+          registry: gcr.io
+          username: _json_key
+          password: ${{ secrets.GOOGLE_CREDENTIALS }}
+
+      # Build docker image
+      - name: Build Docker image
+        if: github.event_name != 'pull_request'
+        id: build-image
+        run: make CONTAINER_REGISTRY='${{ env.CONTAINER_REGISTRY }}' docker-build-flow
+      
+      # Push docker image
+      - name: Push Docker image
```

Comment:
I'm not sure we want to push containers for every PR commit. What's the intention with this CU?

Reply:
Thank you for the review.
The original instructions we received targeted an Observer Microservice in a different repository. I was not sure whether this is still the goal, so I just removed the file based on this review.